### PR TITLE
EPMRPP-36955 React. Admin Part. Server settings page

### PR DESCRIPTION
**Pull request by AmsterGet**
Also some issues with settings components have been fixed in this PR.

### CODE REVIEW

File: app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    defaultMessage: 'Statistics',
+  },
+});
+
+@connect(
+  (state) => ({
+    activeTab: state.location.payload.settingTab,
+  }),
+  {
+    changeTab: (link) => redirect(link),
```
Comment (DiscoElevator) [inline app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx]: changeTab: redirect

File: app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  },
+  statistics: {
+    id: 'ServerSettingsTabs.statistics',
+    defaultMessage: 'Statistics',
+  },
+});
+
+@connect(
+  (state) => ({
+    activeTab: state.location.payload.settingTab,
```
Comment (DiscoElevator) [inline app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx]: activeTab: payloadSelector(state).settingsTab

File: app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  {
+    changeTab: (link) => redirect(link),
+  },
+)
+@injectIntl
+export class ServerSettingsTabs extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    activeTab: PropTypes.string,
+    changeTab: PropTypes.func,
```
Comment (DiscoElevator) [inline app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx]: onChangeTab

File: app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx]: Fixed

File: app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx (hunk 3 above)
Comment (AmsterGet) [reply app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx]: Fixed

File: app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx (hunk 2 above)
Comment (AmsterGet) [reply app/src/pages/admin/serverSettingsPage/serverSettingsTabs/serverSettingsTabs.jsx]: Fixed

**Pull request merged**